gatsby-theme-landing-page/src/components/head.js

```diff
@@ -1,8 +1,15 @@
 import * as React from "react";
 import { Helmet } from "react-helmet";
 
-export default function Head({ title, description, image }) {
+export default function Head({
+  title = "",
+  description = "",
+  image = "",
+  noIndex = false,
```

Comment:
Not needed here (or relevant yet), but it might make sense to pass `children` as a prop – thinking about the API for the layout component/etc., but let's sync up later on that one